19.03.5.sh

```diff
@@ -511,6 +518,32 @@ do_install() {
 			)
 			exit 0
 			;;
+		sles)
+			(
+			set -x
+			$sh_c 'zypper refresh'
+			docker_pkg=$(zypper se -n -s docker | grep -E "\sdocker\s.*${docker_version//\./\\.}.*$architecture" | sort --sort=v -k3 | tail -1 | cut -d '|' -f4 | cut -d ' ' -f2)
+			if [ -z "$docker_pkg" ]; then
+			    trunc_docker_version=${docker_version%.*}
+				available_docker_pkgs=$(zypper se -n -s docker | grep -E "\sdocker\s.*${trunc_docker_version//\./\\.}.*$architecture" | sort --sort=v -k3 | cut -d '|' -f4 | cut -d ' ' -f2 | cut -d '_' -f1)
+				cat >&2 <<-EOF
```

Comment:
Can we rename this to (if this makes sense)

```
No package for docker found that matches requested version $docker_version for $architecture

Packages found for docker ${trunc_docker_version}:
```

Reply:
Updated in latest commit!